content/3-posture/1-compliance/identify_and_remediate.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Identifying and Remediating Compliance Requirements"
+chapter: true
+weight: 1
+---
+
+
+## Identify
+
+Sysdig provides a general overview of your security posture.
+
+Go to Home
+
+Go to Posture > Compliance
+
+## Remediation for CSPM
+
+foo
```

Comment:
foo?

Reply:
Render the current version, the Posture section is not indexed.